docker-gc

```diff
@@ -54,6 +55,9 @@ SYSLOG_LEVEL=${SYSLOG_LEVEL:=info}
 SYSLOG_TAG=${SYSLOG_TAG:=docker-gc}
 DRY_RUN=${DRY_RUN:=0}
 EXCLUDE_DEAD=${EXCLUDE_DEAD:=0}
+REMOVE_VOLUMES=${REMOVE_VOLUMES:=0}
+EXCLUDE_VOLUMES_IDS_FILE=${EXCLUDE_VOLUMES_IDS_FILE:=/etc/docker-gc-exclude-volumes}
+VOLUME_DELETE_ONLY_DRIVER=${VOLUME_DELETE_ONLY_DRIVER:=locale}
```

Comment:
Oh, good question: it should be "local". This was a pretty nasty bug too, because if you would not specify any value for ``VOLUME_DETE_ONLY_DRIVER``` it would not actually remove any volume... It is now fixed.